pkg/template/resource_test.go

```diff
@@ -322,9 +322,15 @@ func Test_ResolveTrigger(t *testing.T) {
 	}{
 		{
 			name: "1 binding",
-			trigger: bldr.Trigger("my-triggertemplate", "v1alpha1",
-				bldr.EventListenerTriggerBinding("my-triggerbinding", "", "my-triggerbinding", "v1alpha1"),
-			),
+			trigger: func() triggersv1.EventListenerTrigger {
```

Comment:
wouldn't it be more readable to just create the EventListenerTrigger object directly instead of the builder here?

```
triggersv1.EventListenerTrigger{
  Bindings: []*triggersv1.EventListenerBinding{{
    Ref: "my-triggerbinding",
  }},
  Template: triggersv1.TriggerTemplate{
    Name: "my-triggertemplate",
  }
}
```

Reply:
Its slightly more verbose but it seems clearer than using a builder to create an EL and then returning the first Trigger from that EL

Reply:
Yes, I changed to that.